common/capitalizeName.js

```diff
@@ -0,0 +1,18 @@
+// capitalize users name
+const capitalizeName = (req, res, next) => {
+  // expects an object with a 'name' field, ie: {name: 'justin'}
+  if (!req.body.name) {
+    res.status(400).json({errorMessage: 'no name in request'});
+  } else {
+    //req.body.name = req.body.name[0].toUpperCase() + req.body.name.slice(1);
+    let name = req.body.name.split(' ');
+    console.log(name);
+    name = name.map(word => word[0].toUpperCase() + word.slice(1));
```

Comment:
Good job breaking this into it's own capitalize name file. Also I like that you took the time to capitalize just the first letter instead of just the whole word. (which I think is the goal)